frameworks/helloworld/tests/test_overlay.py

```diff
@@ -69,20 +69,16 @@ def test_overlay_network():
     assert(len(deployment_plan["phases"][4]["steps"]) == 4)
 
     # Due to DNS resolution flakiness, some of the deployed tasks can fail. If so,
-    # we wait for them to redeploy, but if they don't fail we still want to proceed.
-    try:
-        sdk_plan.wait_for_in_progress_recovery(PACKAGE_NAME, timeout_seconds=60)
-        sdk_plan.wait_for_completed_recovery(PACKAGE_NAME, timeout_seconds=60)
-    except TimeoutExpired:
-        pass
+    # we wait for them to redeploy, but if they can't then we won't proceed as we
+    # expect the tasks to be running for this test.
+    sdk_plan.wait_for_in_progress_recovery(PACKAGE_NAME, timeout_seconds=60)
```

Comment:
Are we really expecting a recovery here? I don't see what what operation before would cause this.